docs/source/checks/vision/performance/plot_single_dataset_scalar_performance.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+"""
+Single Dataset Scalar Performance
+******************************
+
+This notebooks provides an overview for using and understanding single dataset scalar performance check.
+
+**Structure:**
+
+* `What Is the Purpose of the Check? <#what-is-the-purpose-of-the-check>`__
+* `Generate data an model <#generate-data-and-model>`__
+* `Run the check <#run-the-check>`__
+
+What Is the Purpose of the Check?
+=================================
+This check returns a metric result as a single scalar, which is especially useful for monitoring.
```

Comment:
Explain here about the relationship between metric return type and the reduce function.